### ISSUE TITLE
Support the ability to create a schema before creating migration tables

### ISSUE THREAD

**Issue by aikendaniel** [nkonev/r2dbc-migrate#18]
If we are specifying a non-default schema to use and the database is empty, the migration fails because the schema does not exist. It would be great to have r2dbc-migrate create the schema that is specified via `r2dbc.migrate.migrations-schema` if it does not exist.

I observed this behavior when using Postgres and a Spring Boot app. I understand that I can probably use a `SqlR2dbcScriptDatabaseInitializer`, but I think this feature should be baked into the library instead.

**Comment by nkonev** [comment]
@aikendaniel 
I think it can be something like "premigration script", in general it can be creating database, creating user and so on.

**Comment by aikendaniel** [comment]
That works for my use case! Would pointing to a `.sql` file be more flexible for multi-line scripts, though?

**Comment by nkonev** [comment]
> Would pointing to a .sql file be more flexible for multi-line scripts

Interesting suggestion, I will explore how to reuse existing mechanisms to load those scripts.

**Comment by nkonev** [comment]
I've pushed 1.8.6 version. It gonna be available at Maven Central after some time.
TL;DR;

* Use the "premigration" modifier to create schema: `V0__create_schemas__premigration.sql`
* Those scripts should be idempotent, e. g. `CREATE SCHEMA IF NOT EXISTS "my premigrable scheme";`
